### DISCUSSION TITLE
Confidence interval in DFKDE

### DISCUSSION THREAD

**Discussion by JACKYNIKK** [open-edge-platform/anomalib#669] [Q&A]
What is the confidence interval parameter in DFKDE? What does is represnt and influence? Could somebody make an example on how it changes the output of the modelling of a kernel dentity estimation?

**Comment by djdameln** [comment]
Hi, apologies for the delay in responding. Do you mean the `confidence_threshold` parameter that is listed in the default `config.yaml` of the DFKDE model? That parameter has been deprecated and has no function anymore. It should have been removed but it was overlooked. Originally it served as the confidence threshold above which model predictions were labeled as anomalous, but that functionality has now been replaced by Anomalib's thresholding mechanism.

Update: The parameter has been removed, see https://github.com/openvinotoolkit/anomalib/pull/806
